lib/register.php

```diff
@@ -460,6 +460,11 @@ function gutenberg_redirect_to_classic_editor_when_saving_posts( $url ) {
 function gutenberg_link_revisions_to_classic_editor( $url ) {
```

Comment:
Should we update the function documentation to reflect that it only redirects to classic editor when it's intended to?

Reply:
yes, i'll add that.